pepys_import/core/store/postgres_db.py

```diff
@@ -253,6 +253,10 @@ class Participant(BasePostGIS):
 
 
 class Datafile(BasePostGIS):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._measurement_files = []
```

Comment:
@BarisSari - I don't think the entities in this array are `files` - I think they're just `measurements` (state, contact, comment).

Reply:
Yes, you're right. Variable renamed:

https://github.com/debrief/pepys-import/pull/98/commits/814708e9e209aa01ddf534e0873fafd107867be5